rubin_sim/maf/batches/srd_batch.py

```diff
@@ -168,9 +169,9 @@ def astrometryBatch(
     Parameters
     ----------
     colmap : `dict` or None, optional
-        A dictionary with a mapping of column names. Default will use OpsimV4 column names.
+        A dictionary with a mapping of column names.
```

Comment:
In this an other instances, removing statement of what the default is may not be the best idea.
Perhaps just put it on a new line.